bin/setup

```diff
@@ -45,7 +44,9 @@ FileUtils.chdir APP_ROOT do
   puts "\n== Removing old logs and tempfiles =="
   system! 'bin/rails log:clear tmp:clear'
 
-  puts "\n== Restarting application server =="
-  # system! 'bin/rails restart'
-  system! 'bin/rails server -p 8080'
+  unless ARGV.include?('--skip-server')
+    puts "\n== Starting development server =="
+    STDOUT.flush # flush the output before exec(2) so that it displays
+    exec 'bin/dev', '-p', '8080'
```

Comment:
What's the benefit of using `bin/dev` over explicitly using `bin/rails server`?

Reply:
See comment: https://github.com/dradis/dradis-ce/pull/1313#discussion_r2230342200